src/components/Common/OpenPositions.js

```diff
@@ -1,12 +1,10 @@
 import React from 'react'
+import { Padding } from 'styled-components-spacing'
 import { Col, Row } from '../grid'
 import JobsByLocation from '../JobsByLocation'
```

Comment:
Just looking at that layout, this `JobsByLocation` is only used in this this file, can you move this to `Common` as well please